deploy/scripts/src/mobile/utils/test-client.ts

```diff
@@ -5,7 +5,7 @@ import { config } from './config'
 export function postTestClientStart (): Response {
   return http.post(
     buildUrl('start', config.testClientExecuteUrl),
-    JSON.stringify({ frontendUri: config.frontendUrl }),
+    JSON.stringify({ backendUri: `${config.backendUrl}/`, frontendUri: config.frontendUrl }),
```

Comment:
`backendUri` is not the name of the key that needs to be passed in the test client body to change the ApiLocation. The key name should be `target`

Reply:
Passing in `backendUri` here will not override the value in the ApiLocation, which means that the ApiLocation in the response will still use the default non-pretty BE URL. There is an issue with passing this into the test client at the moment. 

When we pass in a `target` which does not contain a `/` at the end, we get a response from the test client, but the ApiLocation is missing a forward slash before verifyAuthorizeRequest, e.g. `https://api-backend-api.review-b.build.account.gov.ukverifyAuthorizeRequest`.

When the `target` does not contain a `/` at the end, there is an error in invoking one of the APIs that the test client lambda calls (I think it's the /.well-known/jwks.json endpoint). 

So, if we include a / we know we'll get an error from the test client. Which means if we want to complete this work without updating the test client we would need to pass in `target: config.backendUrl` to the body here, and tactically edit the ApiLocation so that we insert a / between the `backendUrl` and the `verifyAuthorizeRequest`, or something along these lines. 

Open to other suggestions but this is my understanding of the situation at the moment!